src/NServiceBus.AcceptanceTesting/Support/EndpointRunner.cs

```diff
@@ -173,13 +173,13 @@ void RegisterInheritanceHierarchyOfContextInSettings(ScenarioContext context)
 
         async Task Cleanup()
         {
-            dynamic transportCleaner;
+            IConfigureTestExecution transportCleaner;
             if (busConfiguration.GetSettings().TryGet("CleanupTransport", out transportCleaner))
             {
                 await transportCleaner.Cleanup();
```

Comment:
The amount of :beer: owed to @danielmarbach in Thailand is getting worrying, I might have to chip in and help him drink it! :)